test/core/templaters/jinja_test.py

```diff
@@ -413,6 +413,14 @@ def assert_structure(yaml_loader, path, code_only=True, include_meta=False):
     if parsed is None:
         print(p)
         raise RuntimeError(p[0][1])
+    # Enable this code to create a new test .yml file, i.e. when adding a new
+    # test case.
+    # if parsed:
+    #     r = parsed.as_record(code_only=True, show_raw=True)
+    #     if r:
+    #         with open(path + ".yml", "w") as f:
+    #             import yaml
+    #             yaml.dump(r, f, sort_keys=False)
```

Comment:
As we mentioned on slack it would def be nice to get this included in `generate-fixture-yml` (or it's own script if you prefer). Could you make a small issue and copy this code into it and then remove from this PR so we keep the code here clean and then we can decide where to put this 😄

Reply:
I created an issue: #2247. I'll remove this code.